docs/modules/ROOT/pages/guides/apollo-federation.adoc

```diff
@@ -25,6 +25,9 @@ npm install @apollo/server @neo4j/graphql graphql-tag neo4j-driver
 
 === Example subgraphs
 
+NOTE: In the two subgraphs below, notice that `@shareable` has been applied to both of the `Location` definitions.
```

Comment:
Is this not only necessary for the shared fields not the type? My understanding was that applying `@shareable` to a type, in effect applies it to every field. I could be wrong though

Reply:
Indeed, but that is something we have not yet implemented, to only propagate it to the relevant fields... Baby steps!! 😬

Reply:
I think the documentation feels a bit misleading here though. The reason `@shareable` needs to be on the type, is not because federation requires it to be there, but because we don't _yet_ propagate `@shareable` on a field.

If we did do that then it could just be on the shared fields and having two `Location` types would be fine

Reply:
That's a good point actually, I don't think anything needs to be `@shareable` in this example as are `@key` fields not implicitly shared across the schema